src/routes/conversation/[id]/web-search/+server.ts

```diff
@@ -83,10 +83,8 @@ export async function GET({ params, locals, url }) {
 						let text = "";
 						try {
 							text = await parseWeb(link);
-							appendUpdate(`Browsing webpage: ${link}`);
```

Comment:
I think this was fine (+ @nsarrazin comment)

Reply:
> Minor nit is that the update collapse menu is getting pretty big and I feel like some updates could be combined/removed imo.

from @nsarrazin [here](https://github.com/huggingface/chat-ui/pull/427#pullrequestreview-1624289710)